src/DependencyInjection/Providers/GoogleProviderConfigurator.php

```diff
@@ -19,9 +19,11 @@ public function buildConfiguration(NodeBuilder $node)
         // todo - add the comments as help text, and render in README
         $node
             ->scalarNode('access_type')
+                ->defaultValue('')
                 ->info('Optional value for sending access_type parameter. More detail: https://developers.google.com/identity/protocols/OpenIDConnect#authenticationuriparameters')
             ->end()
             ->scalarNode('hosted_domain')
+                ->defaultValue('')
                 ->info('Optional value for sending hd parameter. More detail: https://developers.google.com/identity/protocols/OpenIDConnect#hd-param')
             ->end()
             ->arrayNode('user_fields')
```

Comment:
I tested without defining default value for `user_fields` and it worked fine. So I suppose arrayNode / prototype make default value useless